python/lsst/analysis/tools/tasks/propertyMapAnalysis.py

```diff
@@ -111,7 +108,7 @@ class PerTractPropertyMapAnalysisConfig(
         itemtype=str,
         doc="Keyword arguments to pass to the colorbar.",
```

Comment:
Here, and elsewhere, I'd recommend adding in a documentation of which keys are invalid. But I also worry that it could go out of sync (and hence my comment preferring `validate` over `keyCheck`)